tensorflow/core/profiler/convert/xplane_to_op_stats_test.cc

```diff
@@ -52,6 +52,9 @@ TEST(ConvertXPlaneToOpStats, PerfEnv) {
 
   XPlaneBuilder device_plane(
       GetOrCreateGpuXPlane(&space, /*device_ordinal=*/0));
+  device_plane.AddStatValue(*device_plane.GetOrCreateStatMetadata(
+                                GetStatTypeStr(StatType::kDevManufacturer)),
+                            std::string("Nvidia"));
```

Comment:
Make this a named constant, use here and elsewhere.

Reply:
Done.